plantio/models/plantio.py

```diff
@@ -15,3 +21,18 @@ class Plantio(models.Model):
     talhao = models.ForeignKey(Talhao, on_delete=models.PROTECT)
     dataPlantio = models.DateField()
     estado = models.CharField(max_length=10, choices=PLANTIO_CHOICES)
+    agrotoxico = models.ManyToManyField(Agrotoxico, through='AssociacaoPlantioAgrotoxico')
+
+
+class AssociacaoPlantioAgrotoxico(models.Model):
+    plantio = models.ForeignKey(Plantio, on_delete=models.PROTECT)
+    agrotoxico = models.ForeignKey(Agrotoxico, on_delete=models.PROTECT)
+    dataAplicacao = models.DateField()
+    # TODO: Implementação de foto (Azure) -> issue 152
+    # caminhoFotoAgrotoxico = models.CharField(max_length=255)
+    dosagemAplicacao = models.DecimalField(
+        max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal('0.01'))])
+    estadoAnalise = models.CharField(max_length=1, choices=ESTADOS_CHOICES, default='A')
```

Comment:
ESTADOS_CHOICES como uma lista de tupla nao fica muito expressivo, acho que daria pra usar um enum ai pra ficar mais legivel.

```suggestion
    estadoAnalise = models.CharField(max_length=1, choices=ESTADOS_CHOICES, default=ESTADOS_CHOICES.ANALISE)
```
![image](https://user-images.githubusercontent.com/54081877/160609613-51a52e00-4a66-4416-9415-4290ced14392.png)

PS: o enum não nessariamente deve ta dentro da classe, pode criar la na pasta de constants mesmo.

Reply:
Não fiz essa alteração pois esse atributo ainda não foi utilziado explicitamente em código (com exceção da model).